server/public/model/config.go

```diff
@@ -3648,6 +3648,21 @@ func (s *ExportSettings) SetDefaults() {
 	}
 }
 
+type AccessControlSettings struct {
+	EnableAttributeBasedAccessControl *bool `access:"write_restrictable,cloud_restrictable`
+	EnableChannelScopeAccessControl   *bool `access:"cloud_restrictable"`
+}
```

Comment:
Maybe we can also add Property group mapping here. It could serve several purposes:
- Decide on which property groups can be used in ABAC
- Create aliases? eg. `user.custom_profile_attributes` can be used as `user.attr` with alias. It could be cleaner in the UI.
Thoughts cc @mgdelacroix

Reply:
I like the idea of having the property groups usable by the feature configurable, but I'd defer adding the configuration parameter until we have multiple groups to avoid confusion.